### PR TITLE
add developer page, developer card widget, and 3 images

### PR DESCRIPTION
**Pull request by ltkien0105**
I completely add developer page to this project. In developer page. I code the UI same as design link in Figma. In addition, I add 3 images include: github.png, linkedin.png and company.png (I don't know exact mean of this image)

### CODE REVIEW

**Comment by Hareesh-Nandigrama** [comment]
@ltkien0105 Thanks for your contribution. I would like to suggest a few changes before merging
1) The github, linkden, and outlook links should be clickable and should redirect to the appropriate page in browser
2) Add the developers page in the side bar with the name team as shown below
image
3) Raise the pull request in the hacktoberfest branch
4) Link the issue this pull request is solving

We will merge your pr once the following changes are completed

**Review by Ashutosh-777 — CHANGES_REQUESTED** [review]
1. Icons should redirect to their website (Currently, Icons you made are just images)
2. go to this location and add the Developer's page to the flow 
    <lib/widgets/home/home_drawer.dart> I have attached a Screenshot for the same in the design file
3. Also, the "Meet the Team" text will be on the top of the Developer's Page, the design for which will be updated by tonight.

**Comment by Ashutosh-777** [comment]
outdated

**Pull request closed without merging**
